components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/RefreshGrantHandler.java

```diff
@@ -282,6 +297,63 @@ private boolean validateRefreshTokenStatus(RefreshTokenValidationDataDO validati
         return true;
     }
 
+    private boolean validateAuthenticatedUser(RefreshTokenValidationDataDO validationBean,
```

Comment:
This method always returns `true` right? And is there any usage of the returned value?

Reply:
These boolean values are not used in the code, only asserted in unit tests. However, for consistency, I have added them since all validation functions are written in the same manner.